src/main/kotlin/ro/hibyte/betting/dto/CompleteBetTypeDto.kt

```diff
@@ -0,0 +1,20 @@
+package ro.hibyte.betting.dto
+
+import ro.hibyte.betting.entity.BetTemplateType
+import ro.hibyte.betting.entity.BetType
+
+data class CompleteBetTypeDto(
+    val id: Long? = null,
+    val name: String,
+    val type: BetTemplateType,
```

Comment:
DTOs should not have the enum type included in them, as the front end can send any string value for this field.

Please replace with string:
```suggestion
    val type: String,
```